### ISSUE TITLE
Surface AA/AAA recommendations

### ISSUE THREAD

**Issue by gregwhitworth** [webhintio/hint#3034]
When doing an assessment and there are accessibility issues, it would be valuable to denote in some manner which items impact failing either AA or AAA. This would be valuable as more and more laws require a specific rating to be passed. And while the documentation does denote this information it would be good to see it while the developer is looking at the results. This could be as simple as a subhead of what category these issues come from.
